client/src/components/framework/toasters.ts

```diff
@@ -7,8 +7,17 @@ import styles from "./toasters.css";
 /** Singleton toaster instance. Create separate instances for different options. */
 let ToastTopCenter: Toaster | null = null;
 
+/**
+ * OverlayToaster.create() in blueprintjs v5 throws a console error of
+ * "uses ReactDOM.render which is being deprecated in React 18"
+ * This is a known issue until blueprintjs v6 is released.
+ * https://github.com/palantir/blueprint/issues/5212#issuecomment-1294958195
+ * using any of the alternative methods invokes the same error.
+ * https://blueprintjs.com/docs/#core/components/toast.overlaytoaster
+ */
 if (typeof document !== "undefined") {
-  ToastTopCenter = Toaster.create({
+  // @ts-expect-error ts-migrate(2339) FIXME: Type 'Promise<Toaster>' is missing the following... Remove this comment to see the full error message
+  ToastTopCenter = OverlayToaster.createAsync({
```

Comment:
This is the recommended method by blueprint and also throws the same console error